modules/assim.batch/R/pda.define.llik.R

```diff
@@ -67,18 +67,21 @@ pda.calc.error <-function(settings, con, model_out, run.id, inputs, bias.terms){
   for (k in seq_len(n.input)) {
     
     if (settings$assim.batch$inputs[[k]]$likelihood == "Laplace") {
```

Comment:
Might also be a way to make them a special case of Betsy's more general Metric function catalog?